sample/MozillaAllHands_17/sponsors

```diff
@@ -0,0 +1,12 @@
+[
+	{
+        "description": "",
+        "level": "1",
+        "url": "https://www.mozilla.org/en-US/firefox/new/",
+        "sponsor_type": "Gold Sponser",
+        "logo": "https://drive.google.com/uc?export=download&id=0B6k_UTO5VOZKQXRDYTQ1aGl2d2c",
```

Comment:
Please change this to local image too. The app is working fine.

Nice job! Just remove the redundant images from the sample, update the logo and change to local image and you are done